contracts/extensions/IssuanceExtension.sol

```diff
@@ -34,8 +34,12 @@ import { IManagerCore } from "../interfaces/IManagerCore.sol";
  * @title IssuanceExtension
  * @author Set Protocol
  *
- * Smart contract global extension which provides DelegatedManager owner and methodologist the ability to accrue and split 
+ * Smart contract global extension which provides DelegatedManager owner and methodologist the ability to accrue and split
  * issuance and redemption fees. Owner may configure the fee split percentages.
+ *
+ * Notes
+ * - the fee split is set on the Delegated Manager contract
+ * - when fees distributed via this contract will be inclusive of all fee types that have already been accrued
```

Comment:
Note the `issuanceExtension` perform a `distributeFee` and **not** an `accrueFeeAndDistribute`. 

So it collects fees previously accrued (like issuance and redemption fees which are accrued on `issue`/`redeem` calls) but does not collect fees pending accrual (like streaming fees).